roles/edpm_ovs/tasks/update.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Update openvswitch packages without stopping the service
+  become: true
+  tags:
+    - update
+    - ovs
+  ansible.builtin.shell: >
+    dnf update -y openvswitch --setopt=tsflags=noscripts
```

Comment:
are we disabling scripts for all packages, not just for openvswitch3.x? Can we skip scriptlets just for the actual ovs package and not for e.g. rhoso-openvswitch that provides the fake 'openvswitch' dependency?